web/pandas/community/ecosystem.md

```diff
@@ -345,11 +345,12 @@ which pandas excels.
 
 ## IO
 
-### [NTV-pandas](https://github.com/loco-philippe/ntv-pandas)*: A semantic, compact and reversible JSON-pandas converter*
+### [NTV-pandas](https://github.com/loco-philippe/ntv-pandas): *A semantic, compact and reversible JSON-pandas converter*
```

Comment:
```suggestion
### [NTV-pandas](https://github.com/loco-philippe/ntv-pandas)
```

We don't add descriptions on the title for any other project, can you please remove it?